src/prost/wrapper_eraftpb.rs

```diff
@@ -0,0 +1,922 @@
+impl Entry {
```

Comment:
Can we have a document comment stating the file is generated by the script, and should not be edited?

Reply:
nrc/kvproto-build#3